src/components/elements/lockup/lockup-a.tsx

```diff
@@ -6,8 +6,10 @@ const LockupA = ({line1, line5, siteName, logoUrl}: FooterLockupProps) => {
   return (
     <div className="py-10">
       <Link href="/" className="no-underline text-black">
-        <div className="flex flex-col items-start lg:flex-row gap-4">
-          <LockupLogo logoUrl={logoUrl} siteName={siteName}/>
+        <div className="flex flex-col lg:flex-row gap-4">
+        <div className="mt-auto">
```

Comment:
Just needs correct indentation and then it's g2g
```suggestion
          <div className="mt-auto">
```